packaging/nnstreamer.spec

```diff
@@ -33,6 +33,7 @@
 %define		caffe2_support 0
 %define		mqtt_support 0
 %define		lua_support 1
+%define		tvm_support 1
```

Comment:
is it available in all gbs profiles?

Reply:
Yes it should be. I'm going to add test cases on armv7/aarch64 soon

Reply:
Should be disabled in da and tv profile?

Reply:
Ah I see. I thought about architectures.. sorry

Reply:
Updated!